src/test/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibraryUnitTest.java

```diff
@@ -19,9 +18,170 @@
 import static org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary.*;
 
 /**
- * Tests for the Wellformed read filter. .
+ * Tests for the read filter library.
  */
-public final class WellformedReadFilterUnitTest extends ReadFilterTest {
+public final class ReadFilterLibraryUnitTest {
+    private static final int DEFAULT_READ_GROUP_COUNT = 5;
+    private static final String DEFAULT_READ_GROUP_PREFIX = "ReadGroup";
+    private static final String DEFAULT_PLATFORM_UNIT_PREFIX = "Lane";
+    private static final String DEFAULT_PLATFORM_PREFIX = "Platform";
+    private static final int DEFAULT_CHROMOSOME_COUNT = 1;
+    private static final int DEFAULT_CHROMOSOME_START_INDEX = 1;
+    private static final int DEFAULT_CHROMOSOME_SIZE = 1000;
+
+    private final int groupCount;
```

Comment:
This should probably be removed and replaced directly with DEFAULT_READ_GROUP_COUNT